src/common.ts

```diff
@@ -167,4 +167,20 @@ export function deleteInstallFile(type: InstallFileType): Promise<void> {
 export function convertNativePathToPosix(pathString: string): string {
     let parts = pathString.split(path.sep);
     return parts.join(path.posix.sep);
+}
+
+/**
+ * This function checks to see if a subfolder is part of folder.
+ * 
+ * Assumes subfolder and folder are absolute paths.
+ * 
+ * @param subfolder subfolder to check if it is part of the folder parameter
+ * @param folder folder to check aganist
+ */
+export function isSubfolderOf(subfolder: string, folder:string): boolean {
```

Comment:
Does this work if the paths differ by case? Does it matter?

Reply:
Nit: space after `folder:`

Reply:
@DustinCampbell  Case sensitivity matters for Linux OS. Should I make it insensitive for Windows and Mac?

Reply:
Not necessarily. If both values are expected to have the correct casing, it shouldn't matter. However, if they come from different sources, it might be a problem.